docs/clients/uavcan_node.rst

```diff
@@ -108,32 +108,34 @@ Type ID 80 | UAVCAN Node
+        | 11     | bypass_arming           | get, set, save | uint8     | :math:`\text{Bool}`             | This setting allows the module to bypass arming with DroneCAN throttle messages. DroneCAN messages will not be impacted by the arming state, will not cause arming state transistions, and will use the zero behavior setting.                                                                                                                                                                                  |
+        +--------+-------------------------+----------------+-----------+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 12     | arming_by_arming_status | get, set, save | uint8     | :math:`\text{Bool}`             | This setting allows the module to arm and disarm based on the DroneCAN ArmingStatus message. Flight controllers are responsible for transmitting the ArmingStatus message, the module is a listener only. If this value is true, your module will arm based on the ArmingStatus message. Otherwise, normal arming applies. You must set :ref:`manual_arming_throttle_source` when arming with ArmingStatus      |
+        +--------+-------------------------+----------------+-----------+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 13     | telemetry_style         | get, set, save | uint8     | :math:`\text{Bool}`             | Setting this to 0 will output the ESC Status and ESC Status Extended as telemetry. Setting this to 1 will outputs the ESC Status and Device Temperature as telemetry.                                                                                                                                                                                                                                           |
```

Comment:
can you link over to where we give more details on this guy? https://iqmotion.readthedocs.io/en/feature-update-client-tables/communication_protocols/dronecan_protocol.html#uavcan-equipment-esc-statusextended-data-type-id-1036